lib/challenge.js

```diff
@@ -19,7 +19,16 @@
   // - park
 
 // Answer Starts Here
-
+'webCal'
+Description: An app to keep track of things via your browser.
+Every 'webCal' has (abstraction):
+  - login
+  - dates (year-month-day)
+  - color codes
+Every 'webCal' can (properties/behaviors):
+  - create/edit event
+  - delete event
+  - invite others to events
```

Comment:
This is a good abstraction for this particular problem - can you think of two others?